ztag/stream.py

```diff
@@ -165,6 +166,26 @@ def take(self, pbout):
     def cleanup(self):
         return self.push(noretry=True)
 
+class _KafkaWorkerProcess(multiprocessing.Process):
+
+    def __init__(self, q, topic, main_producer, cert_producer):
```

Comment:
I'm not sure where you've gotten that from. The documentation does not state that and issues on the project explicitly state not to do this. https://github.com/dpkp/kafka-python/issues/570 states "You should not expect to share a KafkaProducer instance across processes; instead you will need to create a new instance for each process."